test/e2e/storage/testsuites/testdriver.go

```diff
@@ -166,9 +166,9 @@ type DriverInfo struct {
 	InTreePluginName string
 	FeatureTag       string // FeatureTag for the driver
 
-	// Max file size to be tested for this driver
+	// Max file size (on disk) to be tested for this driver
```

Comment:
To make this even more clear, maybe phrase it something like "Maximum single file size supported by this driver"

Reply:
@msau42 
Please take another look.